Dropdown/Dropdown.js

```diff
@@ -319,7 +324,11 @@ const DropdownBase = kind({
 				};
 			});
 		},
-		className: ({css, direction, open, width, title, skin, styler}) => styler.append(`${width}Width`, {hasTitle: Boolean(title), open}, skin === 'silicon' ? classnames(css.dropdownButton, {[css.upDropdownButton]: direction === 'above'}) : {}),
+		className: ({adjusteddirection, css, direction, open, width, title, skin, styler}) => {
+			direction = adjusteddirection ? adjusteddirection : direction;
+			return styler.append(`${width}Width`, {hasTitle: Boolean(title), open}, skin === 'silicon' ? classnames(css.dropdownButton, {[css.upDropdownButton]: direction === 'above'}) : {});
+		},
+		adjusteddirection: ({adjusteddirection}) => `${adjusteddirection} center`,
```

Comment:
Could you please check that we need to compute this `adjusteddireciton` prop?

Reply:
We need to compute this prop.
This `adjstedDirection` prop replaces `direction` prop of `DropdownList` component.
So I matched the form of `adjstedDirection` prop to `direction` prop of `DropdownList` component.
If we don't, the shape of `DropdownList` breaks.

Reply:
I think we don't need to add `adjusteddirection` to computed, we can just pass it like this.
```const popupProps = {'aria-live': null, children, direction: `${adjusteddirection} center`,```

Reply:
I accepted your suggestion.